src/apps/common/hiredis_libevent2.c

```diff
@@ -240,7 +240,7 @@ redis_context_handle redisLibeventAttach(struct event_base *base, char *ip0, int
 	  port=port0;
 
   ac = redisAsyncConnect(ip, port);
-  if (!ac) {
+  if (ac->err) {
```

Comment:
What if ac is NULL?  Checking the `redisAsyncConnect` it looks like it can return NULL in a couple of cases

https://github.com/redis/hiredis/blob/master/async.c#L172

Reply:
You are right. Missed that

Reply:
I addressed this comment - they are not handled one after another